packages/scope-manager/src/ScopeManager.ts

```diff
@@ -26,7 +26,7 @@ interface ScopeManagerOptions {
   globalReturn?: boolean;
   sourceType?: 'module' | 'script';
   impliedStrict?: boolean;
-  ecmaVersion?: number;
+  ecmaVersion?: number | 'latest';
```

Comment:
instead of letting this string trickle all the way down into here such that it needs to be handled differently - I think we should instead convert it to a number at the parser level.

This would be similar to what ESLint does for espree ([they annoyingly do it within eslint itself...](https://github.com/eslint/eslint/blob/80cfb8f858888bddfefd7de6b4ecbf5aabe267bc/lib/linter/linter.js))

Reply:
Handled `latest` in parser, used `1e8` like [acorn](https://github.com/acornjs/acorn/blob/691a31b1e3d6936f4e6991da3d81c1a6b06610cf/acorn/src/options.js#L109) and [`babel`](https://github.com/babel/babel/blob/fdfe97879e77db645f8f42a4739525bc40813d94/eslint/babel-eslint-parser/src/configuration.cjs#L14)